examples/exchange/portfolio/2_StreamAccountPortfolioBankBalances/example.go

```diff
@@ -5,21 +5,20 @@ import (
 	"encoding/json"
 	"fmt"
 
-	"github.com/InjectiveLabs/sdk-go/client/common"
 	exchangeclient "github.com/InjectiveLabs/sdk-go/client/exchange"
 )
 
 func main() {
 	// select network: local, testnet, mainnet
-	network := common.LoadNetwork("testnet", "k8s")
-	exchangeClient, err := exchangeclient.NewExchangeClient(network.ExchangeGrpcEndpoint, common.OptionTLSCert(network.ExchangeTlsCert))
+	//network := common.LoadNetwork("testnet", "k8s")
+	exchangeClient, err := exchangeclient.NewExchangeClient("localhost:9907")
```

Comment:
I guess we should not merge this change, looks only meant for testing

Reply:
reverted this local test example